src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/illegalidentifiername/Example2.java

```diff
@@ -0,0 +1,26 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalIdentifierName">
+      <property name="format"
+        value="(?i)^(?!(record|yield|var|permits|sealed|open|transitive|_)$).+$"/>
+    </module>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.illegalidentifiername;
+
+// xdoc section -- start
+public class Example2 {
+  public static void main(String... args) {
+    int open = 4; // violation
+    Object transitive = "transitive"; // violation
+
+    int openInt = 4; // ok, "open" is part of another word
+    Object transitiveObject = "transitiveObject";
```

Comment:
please review issue descriptin one more time, it has text rules and images and links to good PRs.

All examples should be same/idential by javacode, the only  comment content is changing.

Reply:
@romani ,
We were actually waiting for your suggestion regarding the underscore usage as an identifier as mentioned at https://github.com/checkstyle/checkstyle/pull/14471#issuecomment-1941532448

Reply:
done, now the examples structure differ only by comments

Reply:
Please share and image of diff , I still see difference

Reply:
@romani , 
The examples seem similar by java code.
The difference in lines that you see is because of the extra property in config comment of Example2.java